primitives/rialto/src/lib.rs

```diff
@@ -74,6 +74,24 @@ pub const MAX_SINGLE_MESSAGE_DELIVERY_TX_WEIGHT: Weight = 1_500_000_000;
 /// runtime upgrades.
 pub const MAX_SINGLE_MESSAGE_DELIVERY_CONFIRMATION_TX_WEIGHT: Weight = 2_000_000_000;
 
+/// The length of a session (how often authorities change) on Rialto measured in of number of blocks.
+pub const SESSION_LENGTH: BlockNumber = 4;
+
+/// Re-export `time_units` to make usage easier.
```

Comment:
`time_units` aren't used here, right?

Reply:
No they're not. However, they were part of the old runtime code, and they could be useful in the future if we want to change how some units are defined